### PR TITLE
Creating helm charts for local release.

### PR DESCRIPTION
**Pull request by utka**
This is the last PR for making local release possible. 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: release/gcb/docker_tag_push_lib.sh [hunk 1]
```diff
@@ -126,6 +126,10 @@ function docker_push_images() {
     add_docker_creds "${DST_HUB}"
   fi
 
+  if [ -z "${LOCAL_BUILD+x}" ]; then
```
Comment (howardjohn) [inline release/gcb/docker_tag_push_lib.sh]: looks duplicated from immediately above

File: release/gcb/gcb_lib.sh [hunk 2]
```diff
@@ -20,6 +20,7 @@
 #shellcheck disable=SC2164
 SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
 # shellcheck source=release/gcb/docker_tag_push_lib.sh
+#shellcheck disable=SC1091
```
Comment (howardjohn) [inline release/gcb/gcb_lib.sh]: Should either remove line 22 or remove this line as they are both trying to fix the same thing

File: release/gcb/local.sh [hunk 3]
```diff
@@ -37,8 +36,20 @@ popd || return
...
   create_manifest_check_consistency "${artifacts}/manifest.txt"
-  make_istio "${artifacts}" "${DOCKER_HUB}" "${DOCKER_HUB}" "${NEW_VERSION}" "${BRANCH}"
+  make_istio "${artifacts}" "${DOCKER_HUB}" "${DOCKER_HUB}" "${NEW_VERSION}" "${CNI_BRANCH}"
 popd || return
 
 docker_tag_images  "${DOCKER_HUB}" "${NEW_VERSION}" "${artifacts}"
 docker_push_images "${DOCKER_HUB}" "${NEW_VERSION}" "${artifacts}"
+
+pushd "${artifacts}" || exit
+  fix_values_yaml "${NEW_VERSION}" "${DOCKER_HUB}"
```
Comment (howardjohn) [inline release/gcb/local.sh]: This calls gsutil it looks like?

File: release/gcb/gcb_lib.sh [hunk 4]
```diff
@@ -142,3 +143,94 @@ function make_istio() {
...
+  fi
+  local tarball_name="$1"
+  local VERSION="$2"
+  local DOCKER_HUB="$3"
+  if [[ ${tarball_name} == *.zip ]]; then
+    local unzip_cmd="unzip -q"
+    local zip_cmd="zip -q -r"
+  else
+    local unzip_cmd="tar -zxf"
+    local zip_cmd="tar -zcf"
```
Comment (howardjohn) [inline release/gcb/gcb_lib.sh]: maybe not in this pr, but shouldn't we do this before tarring and building mac/linux/windows builds?

File: release/gcb/local.sh (hunk 3 above)
Comment (utka) [reply release/gcb/local.sh]: No, it is function in gcb_lib.sh file line 146

File: release/gcb/gcb_lib.sh (hunk 4 above)
Comment (utka) [reply release/gcb/gcb_lib.sh]: Will look into it.

File: release/gcb/local.sh (hunk 3 above)
Comment (howardjohn) [reply release/gcb/local.sh]: but that function calls gsutil I think

**Comment by howardjohn** [comment]
@utka looks like the shellcheck changes didn't work, its annoying

**Pull request merged**